test/sequential/test-debugger-backtrace.js

```diff
@@ -20,17 +20,20 @@ const path = require('path');
     throw error;
   }
 
-  return cli.waitForInitialBreak()
-    .then(() => cli.waitForPrompt())
-    .then(() => cli.stepCommand('c'))
-    .then(() => cli.command('bt'))
-    .then(() => {
+  async function onWaitForInitialBreak() {
```

Comment:
This function name is not accurate.  Maybe `runTest` or something like that instead?

Reply:
```suggestion
  async function runTest() {
```